client/components/NewEntryForm.js

```diff
@@ -67,6 +83,7 @@ class NewEntryForm extends Component {
               type="text"
               onChange={this.handleChange}
               value={this.state.content}
+              inputProps={{ maxLength: 5000 }}
```

Comment:
this is the same value as characterLimit, can you substitute the variable here?

Reply:
updated to this.state.characterLimit